specification/compute/resource-manager/readme.go.md

```diff
@@ -2,29 +2,20 @@
 
 These settings apply only when `--go` is specified on the command line.
 
-```yaml $(go) && !(track2)
+```yaml $(go) && !$(track2)
 go:
   license-header: MICROSOFT_MIT_NO_VERSION
   clear-output-folder: true
 
 directive:
-    # dynamically add a DummyOrchestrationServiceName value to the enum 
+    # dynamically add a DummyOrchestrationServiceName value to the enum to avoid the generator generates the code by hard-coding the single entried enum value
   - from: compute.json
     where: $..enum
     transform: >-
       if( $.length === 1 && $[0] === "AutomaticRepairs") { 
         $.push('DummyOrchestrationServiceName');
       }
       return $;
-
-  - from: source-file-go
```

Comment:
So we don't need this any more for track 1?

Reply:
We do not need this anymore. Once the enum entry was added into the modeler, the generator can handle it properly. We do not need replace the generated code anymore.

Reply:
![image](https://user-images.githubusercontent.com/85789076/126114888-d0d1ec0d-1683-43dc-956d-9e31d730f971.png)

Reply:
![image](https://user-images.githubusercontent.com/85789076/126538972-d8b1961e-f47e-40f6-9305-866b253c2615.png)

Reply:
![image](https://user-images.githubusercontent.com/85789076/126539008-05fc3dfc-7ff7-4898-9942-c1bc8e9daca2.png)